### PR TITLE
2-css-SerhiiNSN

### PR DESCRIPTION
**Pull request by SerhiiNSN**

### CODE REVIEW

File: homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+                        <li class="menu__list">
+                            <a href="index.html" class="menu__link">About</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main class="main">
+            <section class="games">
+                <div class="games__container">
```
Comment (anna-vrana-moc) [inline homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html:34]: If you have several simular elements - it's a list. So, it is a list of cards. Please update the structure

**Comment by anna-vrana-moc** [comment]
Hi @SerhiiNSN You have one small issue before approving. Go ahead and fix it!

File: homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html (hunk 1 above)
Comment (SerhiiNSN) [reply homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html:34]: I have added the list, waiting for verification)

Comment (anna-vrana-moc) [reply homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html:34]: It seems there might be a misunderstanding. What I meant is that you have multiple similar cards; it's essentially a list of cards. 

For ex. :
```
<ul class="games__list">      //   (games__body - in your code)
   <li class="games__card">
       All card here
       ....
   </li>
</ul>
```
   
 The card content might be:
` <p><span class="bold"> Genre </span> some text </p>`

**Comment by anna-vrana-moc** [comment]
Unfortunately, you created the list in the wrong place. Here, there should only be a list of cards.

File: homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html (hunk 1 above)
Comment (SerhiiNSN) [reply homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html:34]: I hope I have understood you correctly now)

**Review by anna-vrana-moc — APPROVED** [review]

**Comment by anna-vrana-moc** [comment]
Perfect! Approved)

**Pull request merged**